website/content/docs/discovery/services.mdx

```diff
@@ -202,51 +177,49 @@ for the available configuration options.
 
 ### Connect
 
-The `kind` field is used to optionally identify the service as a [Connect
-proxy](/docs/connect/proxies) instance. This value may be "connect-proxy" for
-[Connect-capable](/docs/connect) proxies representing another service,
-"mesh-gateway" for instances of a [mesh gateway](/docs/connect/mesh-gateway),
-"terminating-gateway" for instances of a [terminating gateway](/docs/connect/terminating-gateway),
-or "ingress-gateway" for instances of a [ingress gateway](/docs/connect/ingress-gateway).
-For typical non-proxy instances the `kind` field must be omitted. The `proxy` field
-is also required for Connect proxy registrations and is only valid if `kind` is
-`connect-proxy`. The only required `proxy` field for a `connect-proxy` is `destination_service_name`.
-For more detail please see [complete proxy configuration
-example](/docs/connect/registration/service-registration#complete-configuration-example)
-
--> **Deprecation Notice:** From version 1.2.0 to 1.3.0, proxy destination was
-specified using `proxy_destination` at the top level. This will continue to work
-until at least 1.5.0 but it's highly recommended to switch to using
-`proxy.destination_service_name`.
-
-The `connect` field can be specified to configure
-[Connect](/docs/connect) for a service. This field is available in
-Consul 1.2.0 and later. The `native` value can be set to true to advertise the
-service as [Connect-native](/docs/connect/native). The `sidecar_service`
-field is an optional nested service definition its behavior and defaults are
-described in [Sidecar Service
-Registration](/docs/connect/registration/sidecar-service). If `native` is true,
-it is an error to also specify a sidecar service registration.
-
--> **Deprecation Notice:** From version 1.2.0 to 1.3.0 during beta, Connect
-supported "Managed" proxies which are specified with the `connect.proxy` field.
-[Managed Proxies are deprecated](/docs/connect/proxies/managed-deprecated)
-and the `connect.proxy` field will be removed in a future major release.
+The `kind` parameter determines the service's role. Services can be configured to perform several roles, but you must omit the `kind` parameter for typical non-proxy instances.
+Refer to the [Configuration Entries](/docs/connect/config-entries) documentation for Consul Connect for information about other registration types.
```

Comment:
Thanks for clarifying. I called these values out and linked to additional information. I moved the reference to the Configuration Entries page to the end of the topic so that it would be clear that there were other configuration kinds without introducing confusion.